plasmapy/formulary/paschen.py

```
@@ -0,0 +1,279 @@
+            "W": None,
+            "Fe": 0.059,
+        },
+        "H2": {
+            "Al": 0.095,
+            "Cu": 0.05,
+            "Ni": 0.053,
+            "Pt": 0.02,
+            "C": 0.014,
+            "W": None,
+            "Fe": 0.061,
+        },
+        "He": {
+            "Al": 0.021,
+            "Cu": None,
+            "Ni": 0.015,
+            "Pt": 0.01,
+            "C": None,
+            "W": None,
+            "Fe": 0.015,
+        },
+        "Ne": {
+            "Al": 0.053,
+            "Cu": 0.02,
+            "Ni": 0.031,
+            "Pt": 0.023,
+            "C": None,
+            "W": 0.045,
+            "Fe": 0.022,
+        },
+        "Ar": {
+            "Al": 0.12,
+            "Cu": 0.058,
+            "Ni": 0.058,
+            "Pt": 0.058,
+            "C": None,
+            "W": None,
+            "Fe": 0.058,
+        },
+        "Kr": {
+            "Al": None,
+            "Cu": None,
+            "Ni": None,
+            "Pt": None,
+            "C": None,
+            "W": None,
+            "Fe": None,
+        },
+        "Xe": {
+            "Al": None,
+            "Cu": None,
+            "Ni": None,
+            "Pt": None,
+            "C": None,
+            "W": None,
+            "Fe": None,
+        },
+    }
+
+    # Check if the asked gas and electrode material is supported
+    resg = gas in gases
+    rese = electrode in materials
+
+    # If the gas is supported get the constants A and B
+    if resg:
+        A = paschen_constants[gas][0]
+        B = paschen_constants[gas][1]
+
+        #   Get the townsend_gamma coefficient for the the gas/electrode combination
+        if rese:
+            gam = townsend_gamma[gas]
+            gn = gam[electrode]
+            #       Test if townsend_gamma exists for the demanded gas/electrode configuration
+            #       If not a default townsend_gamma value of 0.01 is taken
+
+            if gn is None:
+                gn = 0.01
```

I have literally no expertise with this area. Is 0.01 a sensible value "in general"? Perhaps, rather than assume it, we could raise a `KeyError: no data for the given gas/electrode`, or at the very least throw a warning and say that we're assuming the default. I think I'd prefer the former approach.